### ISSUE TITLE
Open HTML file instead of navigating to a URL

### ISSUE THREAD

**Issue by ako39f** [chromedp/chromedp#788]
Is it possible for ChromeDB to open an HTML file from disk instead of navigating to a URL?

**Comment by ZekeLu** [comment]
Yes. Check the tests in the source code. There are many examples. You can search for the calls to `testAllocate`.

https://github.com/chromedp/chromedp/blob/5b511f121cb2fc1e868b8667049d3cff32a6e0b1/chromedp_test.go#L118
